doc/sphinx/addendum/micromega.rst

```diff
@@ -54,6 +54,11 @@ or only for reals by ``Require Import Lra``.
 
    This :term:`flag` (set by default) instructs :tacn:`nra` to cache its results in the file `.nra.cache`
 
+.. flag:: Psatz Conversion
+
+   When set, use :term:`definitional equality` (:term:`convertibility <convertible>`) instead of
+   syntactic equality to detect equal terms in the :tacn:`lia`, :tacn:`nia`, :tacn:`lra`,
+   :tacn:`nra` and :tacn:`psatz` tactics during syntaxification.  The default is unset.
```

Comment:
I have no idea what "syntaxification" means.  Can you use another term/phrase or explain the term?  Same thing in the changlog.

Reply:
Thanks for the suggestions.

The current version use "parsing of the goal" instead.

Reply:
Sure thing.

"parsing of the goal" is an improvement, but perhaps we can do better.  IIUC the goal is already in an internal AST format, so "parsing" seems imprecise.  This sounds like an initial step in which the tactics analyze the goal and extract details that are used later on.  Perhaps you could very briefly say what info is extracted, maybe wording like "during the initial step that extracts XXX from the goal."  or "analyzes YYY in the goal".